terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -128,10 +128,17 @@ data "aws_iam_policy_document" "trust-relationship-policy" {
   }
 }
 
-data "aws_iam_policy_document" "execution-combined-policy" {
+data "aws_iam_policy_document" "execution-combined-policy-doc" {
   source_policy_documents = concat([data.aws_iam_policy_document.SSM-Automation-Policy.json, data.aws_iam_policy_document.execution-policy.json])
 }
 
+resource "aws_iam_policy" "execution-combined-policy" {
```

Comment:
Are the cloudformation permissions needed? (in the json above, sorry didn't review this one before :) )